docs/labs/docker-plugin.rst

```diff
@@ -24,7 +24,7 @@ As a user of Docker, it means you can use Flocker directly via:
 
 See the `Docker documentation on volume plugins`_.
 
-This currently depends on the `experimental build of Docker <https://github.com/docker/docker/tree/master/experimental>`_.
+This depends on the Docker 1.8.
```

Comment:
"the Docker 1.8". I guess you mean "Docker 1.8" or maybe "Docker 1.8 or later". This should be consistent with whatever is written in the "Install Docker" section.